android-record/pom.xml

```diff
@@ -0,0 +1,62 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <!-- This module was also published with a richer model, Gradle metadata,  -->
+    <!-- which should be used instead. Do not delete the following line which  -->
+    <!-- is to indicate to Gradle or any Gradle module metadata file consumer  -->
+    <!-- that they should prefer consuming it instead. -->
+    <!-- do_not_remove: published-with-gradle-metadata -->
+    <parent>
+        <groupId>com.fasterxml.jackson.module</groupId>
+        <artifactId>jackson-modules-base</artifactId>
+        <version>2.16.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>jackson-module-android-record</artifactId>
+    <name>Jackson module: Android Record Support</name>
+    <packaging>bundle</packaging>
+
+    <description>Support deserialization into records on Android</description>
+    <url>https://github.com/FasterXML/jackson-modules-base</url>
+
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <properties>
+        <!-- Generate PackageVersion.java into this directory. -->
+        <packageVersion.dir>com/fasterxml/jackson/module/androidrecord</packageVersion.dir>
+        <packageVersion.package>com.fasterxml.jackson.module.androidrecord</packageVersion.package>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
```

Comment:
Could you add jackson-core as a dependency? It is better not to rely on it transitively when you have code that references jackson-core classes.

Reply:
Done. Should I add a dependency on jackson-annotations as well?

Reply:
If there is direct dependency to one or more annotations classes, yes.

Reply:
Done as well.
Actually, the module itself depends directly on jackson-databind and jackson-annotations only. Test code depends on all three. So should I mark the jackson-core dependency "<scope>test</scope>"?

Reply:
Hmmh. No, I think either only include jackson-databind and jackson-annotations deps, or all 3, but not in "test" scope. Either would be fine.

Reply:
Ok, I'll leave all three.